frontend/src/pages/Graphing/GraphingEditor.tsx

```diff
@@ -79,6 +79,8 @@ import { useRetentionPresets } from '@/components/Search/SearchForm/hooks'
 type BucketBy = 'None' | 'Interval' | 'Count'
 const BUCKET_BY_OPTIONS: BucketBy[] = ['None', 'Interval', 'Count']
 
+const MAX_BUCKET_SIZE = 100
```

Comment:
Do we want to up this? For users grouping by unique user or session, they would be limited to 100 values on the graph - maybe this is okay and we should write exceptions for users and session groups?

Reply:
IMO I feel like trying to graph so many buckets is a code smell since it creates charts with tons of small data points, and users should instead be pointed toward using metric drilldown instead, e.g. if they need to see each user contributing to the user count. Maybe there's stuff we could do to improve or customize the drilldowns.